client/parser/config.go

```diff
@@ -153,15 +153,14 @@ func configPush(argv []string) error {
 Sets environment variables for an application.
 
 The environment is read from <path>. This file can be read by foreman
-to load the local environment for your app.
+to load the local environment for your app. The file should be piped via
+stdin, for example 'deis config:push < .env'
 
 Usage: deis config:push [options]
 
 Options:
   -a --app=<app>
     the uniquely identifiable name for the application.
-  -p <path>, --path=<path>
```

Comment:
There isn't any technical reason we can't. I figured it would be confusing to offer two ways to do the same thing, but I don't have any strong feeling either way.